rust/connlib/snownet/src/node.rs

```diff
@@ -2204,7 +2213,7 @@ fn apply_default_stun_timings(agent: &mut IceAgent) {
 }
 
 fn apply_idle_stun_timings(agent: &mut IceAgent) {
-    agent.set_max_stun_retransmits(4);
+    agent.set_max_stun_retransmits(2);
```

Comment:
@conectado I had to drop the max number of retransmissions during IDLE connections. Now that boringtun is pure (:partying_face:), it will actually correctly send the retransmit packets etc in the test.

boringtun will expire a connection after 3 minutes if it hasn't received a packet and it logs that on `ERROR` level. With an interval of 60s, our ICE timeout was 4 minutes on an IDLE connection and thus, after the client has roamed, a gateway would first run into the boringtun ERROR before it would cut the connection due to the ICE timeout.

Reply:
I ended up reverting this and instead lowering the log-level: https://github.com/firezone/boringtun/pull/15.